app/components/example_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ExampleComponent < ViewComponent::Base
```

Comment:
Actually, command `rails generate component Example title --sidecar` creates the spec for the component (see https://viewcomponent.org/guide/#quick-start) but I deleted it since I had no idea which kind of content I should put into (it's empty by default). 
Will recreate it, and try to make it useful.

Reply:
Some example done by commit: 00b92a51e
Very simple at the moment, but it's a start, and only an example.

Reply:
We'll be able to start adding really great unit tests for all UI elements as we move to rendering with `ViewComponent`.

Yay :tada::partying_face:

Reply:
And that means we can start kicking a lot of stuff further down the testing pyramid :muscle: